application/models/models.go

```diff
@@ -33,6 +36,11 @@ func init() {
 		log.Fatal(err)
 	}
 	pop.Debug = env == "development"
+
+	// Just make sure we can use the crypto/rand library on our system
+	if _, err = rand.Read(RandomBytes); err != nil {
+		log.Fatal(fmt.Errorf("error using crypto/rand ... %v", err))
+	}
```

Comment:
I'm not clear on how often `init()` is executed, but isn't it on every API request? If so, why do this every time? `RandomBytes` wouldn't have to be a global if this wasn't here.

Reply:
It only gets run the first time the package is imported.  I was thinking that if it was going to get an error ever, it would get it every time. So, why not catch the error at the start and then not worry about checking for errors later.